plugin/Model/FormModelBuilder/FormModelBuilder.php

```diff
@@ -88,13 +88,17 @@ protected function readFieldnames($module, bool $withInactive = false)
 		try {
 			$this->_pFieldnames->loadLanguage();
 			$fieldnames = $this->_pFieldnames->getFieldList($module);
-
+			$typeSupport = ['integer', 'float', 'varchar', 'text', 'date', 'datetime', 'boolean'];
```

Comment:
This looks legit, but is it needed here? Feels like it should be added for both Fieldnames and FieldsCollectionBuilders, but in a separate project